shared/src/casper_address.rs

```diff
@@ -190,4 +203,32 @@ mod tests {
             Err(String::from("Unsupport Key type."))
         );
     }
-}
\ No newline at end of file
+
+    #[test]
+    fn test_casper_address_account_hash_to_odra_address_conversion_() {
+        let casper_address = CasperAddress::from(mock_account_hash());
+        test_casper_address_to_odra_address_conversion(casper_address);
+    }
+
+    #[test]
+    fn test_casper_address_contract_package_hash_to_odra_address_conversion_() {
+        let casper_address = CasperAddress::from(mock_contract_package_hash());
+        test_casper_address_to_odra_address_conversion(casper_address);
+    }
+
+    fn test_casper_address_to_odra_address_conversion(casper_address: CasperAddress) {
```

Comment:
Why this test is missing #[test]?

Reply:
It's not a test, it's a helper function.